### DISCUSSION TITLE
Paid pro but it doesn't work

### DISCUSSION THREAD

**Discussion by shanematte** [vercel/community#5685] [Help] [answered]
### Summary

Have now paid for a pro account. But domains also write error 402: PAYMENT_REQUIRED. Can you please solve it, we have work idle and clients waiting :(

### Steps to Reproduce

There was a delay but made the payment. What do we need to give you now to resolve the problem? 
account REMOVED  project name "Apartments Project Team"

**Comment by swarnava** [comment, marked as answer]
Hi there,

I can see that your website is working now. Can you confirm?

**Reply by shanematte** [reply]
Yeah, it's working. Thanks
